package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
-  "repository": "github.com:3scale/system.git",
+  "repository": "github.com:3scale/porta.git",
```

Comment:
Can't we use https instead?

Reply:
Sure, I'll push that and see how circleci does this time